### ISSUE TITLE
automatically saving position + different reader options

### ISSUE THREAD

**Issue by ThiccTomato** [dstark5/Openlib#80]
the app should automatically save where im at. it sometimes does but doesnt most of the time. also a top to bottom infinite scrolling reader rather than a left to right swipe reader would be nice to have as an option

**Comment by dstark5** [comment]
Now I'm using a Epub reader package. In upcoming version I will change it to a new Epub reader.

**Comment by shuvashish76** [comment]
Actually I don't like the reader feature... this project should focus on "downloader" & similar functionality.
But since we're already using it as reader can we use other engines for e.g. [KOReader](https://github.com/koreader/koreader) uses both [coolreader engine](https://github.com/koreader/crengine) (their own fork) & [MuPDF](https://mupdf.com/) engine. Similarly [Librera](https://github.com/foobnix/LibreraReader) uses MuPDF engine as well. BTW MuPDF supports PDF, XPS, OpenXPS, CBZ, EPUB, and FB2 etc but KOReader use it for specifically PDF & for spun they use coolreader engine.

**Comment by dstark5** [comment]
> Actually I don't like the reader feature... this project should focus on "downloader" & related functionality not as reader, book tracker/organizer, eBook converter etc... But since we're already using it as reader can we use other engines for e.g. [KOReader](https://github.com/koreader/koreader) uses both [coolreader engine](https://github.com/koreader/crengine) (their own fork) & [MuPDF](https://mupdf.com/) engine. Similarly [Librera](https://github.com/foobnix/LibreraReader) uses MuPDF engine as well. BTW MuPDF supports PDF, XPS, OpenXPS, CBZ, EPUB, FB2 etc but KOReader use it for specifically PDF & for epub & other reflowable files they use coolreader engine.

Yes I should migrate from current Epub reader to a better reader engine like MuPDF. Will work on it.
Thank you
